Readme.md

```diff
@@ -16,7 +16,7 @@ Orchard Core represents a major evolution of the Orchard design principles, and
 
 This work is in its initial design phases. There's a lot of work to do, and yes, we do welcome participation in any shape or form.
 
-The work will focus at first on porting a [minimum viable feature set](https://github.com/OrchardCMS/OrchardCore.Commerce/issues/3).
+The work will focus at first on porting a [minimum viable feature set](https://github.com/OrchardCMS/OrchardCore.Commerce/milestone/1).
 
```

Comment:
In the future it would be good to have more feature documentation, maybe some video tutorials, [our readthedocs page](https://commerce.orchardcore.net/en/latest/) is practically empty. I've created a new issue for it [here](https://github.com/OrchardCMS/OrchardCore.Commerce/issues/249).